<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TDynamicImageDialog</class>
 <widget class="QDialog" name="TDynamicImageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Synamic Image</string>
  </property>
  <property name="locale">
   <locale language="English" country="AmericanSamoa"/>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="labelName">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEditName"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelProtocol">
     <property name="text">
      <string>Protocol</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>HTTP</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>HTTPS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RTP</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RTSP</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelHost">
     <property name="text">
      <string>Host</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEditHost"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelPath">
     <property name="text">
      <string>Path</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEditPath"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelFile">
     <property name="text">
      <string>File</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineEditFile"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelUser">
     <property name="text">
      <string>User</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="lineEditUser"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="labelPassword">
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="lineEditPassword">
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="labelRefreshRate">
     <property name="text">
      <string>Refresh rate</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QSpinBox" name="spinBoxRefreshRate"/>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QCheckBox" name="checkBoxRefreshStart">
     <property name="text">
      <string>Refresh only at startup</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditName</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>lineEditHost</tabstop>
  <tabstop>lineEditPath</tabstop>
  <tabstop>lineEditFile</tabstop>
  <tabstop>lineEditUser</tabstop>
  <tabstop>lineEditPassword</tabstop>
  <tabstop>spinBoxRefreshRate</tabstop>
  <tabstop>checkBoxRefreshStart</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TDynamicImageDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TDynamicImageDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
